force-app/main/default/classes/ProgramSelector_TEST.cls

```diff
@@ -40,6 +42,68 @@ public with sharing class ProgramSelector_TEST {
         }
     }
 
+    @IsTest
+    private static void testGetActiveProgramCohortsFromProgramId() {
+        String methodName = 'getBuckets';
+        List<String> buckets = new List<String>{ ACTIVE_BUCKET };
+        Schema.SObjectType programCohortSObjType = ProgramCohort__c.SObjectType;
+        Schema.SObjectField statusField = ProgramCohort__c.Status__c;
+
+        TestDataFactory.generateProgramData();
+        Id programId = TestDataFactory.programs[0].Id;
+
+        List<Bucket__mdt> cohortStatusBuckets = createBuckets();
+
+        //Insert a cohort with different status so we check that we return
```

Comment:
Should we add something like this to the previous test to ensure we are getting all records back if there is no bucket?